parser/helper.h

```diff
@@ -43,6 +43,13 @@ class MK {
         return std::make_unique<parser::Hash>(loc, kwargs, move(pairs));
     }
 
+    /*
+     * Create a constant node that is already resolved to a symbol.
+     */
+    static std::unique_ptr<parser::Node> ResolvedConst(core::LocOffsets loc, core::SymbolRef symbol) {
+        return std::make_unique<parser::ResolvedConst>(loc, symbol);
+    }
```

Comment:
non-blocking, but: this particular function seems like it's not much additional benefit over inlining `make_unique<ResolvedConstant>` into the few places it's used in this file.

To shorten things, remember that since you're already in the `parser` namespace, nothing in this file needs to be prefixed with `parser::`.

Reply:
Removed 👍